sycl/include/CL/__spirv/spirv_ops.hpp

```diff
@@ -86,6 +86,12 @@ __spirv_JointMatrixSUMadINTEL(
     __spv::__spirv_JointMatrixINTEL<T3, M, N, LC, S> *C,
     __spv::Scope::Flag Sc = __spv::Scope::Flag::Subgroup);
 
+template <typename T, std::size_t R, std::size_t C,
+          __spv::MatrixLayout L = __spv::MatrixLayout::RowMajor,
+          __spv::Scope::Flag S = __spv::Scope::Flag::Subgroup>
+extern SYCL_EXTERNAL __spv::__spirv_JointMatrixINTEL<T, R, C, L, S> *
+__spirv_JointMatrixFillINTEL(const T &v, __spv::Scope::Flag Sc = S);
```

Comment:
```suggestion
__spirv_CompositeConstruct(const T &v);
```
Note, there is not Scope flag for the instruction, but it should be OK, since it presence in the result type.

Reply:
```suggestion
__spirv_JointMatrixFillINTEL(const T v, __spv::Scope::Flag Sc = S);
```
Like const T& the compiler will emit a temporary reference of value T, while we want to pass a constant expression to the instructions below and not load from this temporary reference.